src/ui/components/Library/SidebarFooter.tsx

```diff
@@ -6,7 +6,7 @@ import { AvatarImage } from "../Avatar";
 
 function SidebarFooter({ setModal }: PropsFromRedux) {
   const { user } = useAuth0();
-  const { name, picture } = user!;
+  const { name, picture } = user || { name: "", picture: "" };
```

Comment:
this feels like `useAuth0` should have some better defaults

Reply:
I think we're being optimistic here since the auth0 does mark `user` as possibly undefined. I think it's reasonable to handle this `undefined` here rather than receiving an empty or partial object and inferring the loading state from that.